packages/react-docs/src/pages/components/checkbox.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ComponentDocs from '../../components/componentDocs';
+import PropTypes from 'prop-types';
+import { Checkbox } from '@patternfly/react-core';
+import Example from '../../components/example';
+
+const propTypes = {
+  data: PropTypes.any.isRequired
+};
+
+class CheckboxDocs extends React.Component {
+  state = {
+    checked: true
+  };
+
+  handleCheckboxChange = checked => {
+    this.setState({ checked });
+  };
+
+  render() {
+    const { data } = this.props;
+    const { checked } = this.state;
+
+    return (
+      <ComponentDocs data={data}>
+        <Example title="Controlled checkbox example">
+          <Checkbox
+            checked={checked}
+            onChange={this.handleCheckboxChange}
+            aria-label="controlled checkbox example"
```

Comment:
I think you should add the a prop to the Checkbox component for the aria label.
@jgiardino  Should it be required?

Reply:
aria-label is good.  Other components will move to it.